### PR TITLE
Upgrade packages to track upstream

### PR DESCRIPTION
**Pull request by mseri**
This PR has been generated with a new `xs-opam-upgrade.py` tool, that will be added to `xs-opam` in a separate PR.

### CODE REVIEW

File: packages/upstream-extra/fileutils.0.5.1/findlib [hunk 1]
```diff
@@ -1 +0,0 @@
-fileutils
```
Comment (edwintorok) [inline packages/upstream-extra/fileutils.0.5.1/findlib]: I was not sure what purpose they still serve, but looks like OPAM2 no longer has code to deal with them and they got dropped from upstream repo as well so its ok: https://github.com/ocaml/opam-repository/pull/9328

Comment (mseri) [reply packages/upstream-extra/fileutils.0.5.1/findlib]: Also the current `opam` has not had code for them since years

**Comment by mseri** [comment]
The tool prepared the code but did not make the PR (for now).
It is generating the diff for me to see, so I agree that it should be part of the commit and PR message.
I'll update it

**Comment by mseri** [comment]
Changes:

base64.2.1.2 -> base64.2.2.0
biniou.1.0.13 -> biniou.1.2.0
cmdliner.1.0.0 -> cmdliner.1.0.2
cppo.1.6.0 -> cppo.1.6.1
easy-format.1.2.0 -> easy-format.1.3.1
fmt.0.8.0 -> fmt.0.8.5
hex.1.1.1 -> hex.1.2.0
io-page-unix.2.0.0 -> io-page-unix.2.0.1
io-page.2.0.0 -> io-page.2.0.1
lwt.3.1.0 -> lwt.3.2.1
mirage-block-unix.2.8.3 -> mirage-block-unix.2.9.0
mirage-types-lwt.3.0.5 -> mirage-types-lwt.3.0.7
mirage-types.3.0.5 -> mirage-types.3.0.7
ocaml-migrate-parsetree.1.0.3 -> ocaml-migrate-parsetree.1.0.7
ocamlfind.1.7.3 -> ocamlfind.1.7.3-1
ocamlmod.0.0.8 -> ocamlmod.0.0.9
octavius.1.1.0 -> octavius.1.2.0
omd.1.3.0 -> omd.1.3.1
ounit.2.0.0 -> ounit.2.0.7
ppx_cstruct.3.1.1 -> ppx_cstruct.3.2.1
ppx_deriving.4.2 -> ppx_deriving.4.2.1
ppx_tools_versioned.5.0.1 -> ppx_tools_versioned.5.1
spawn.v0.10.0 -> spawn.v0.10.1
ssl.0.5.3 -> ssl.0.5.5
topkg.0.9.0 -> topkg.0.9.1
uchar.0.0.1 -> uchar.0.0.2
uri.1.9.4 -> uri.1.9.6
xenstore.1.3.0 -> xenstore.2.0.0
xenstore_transport.0.9.4 -> xenstore_transport.1.0.0
yojson.1.3.3 -> yojson.1.4.0

**Pull request merged**